ceph-ansible-pr-syntax-check/build/build

```diff
@@ -50,8 +50,7 @@ function group_vars_check {
 function test_sign_off {
   git fetch
   # shellcheck disable=SC2154
-  test "$(git log --oneline --no-merges origin/"${ghprbTargetBranch}"..HEAD | wc -l)" -ne "$(git log --no-merges origin/"${ghprbTargetBranch}"..HEAD | grep -c Signed-off-by)" && echo "One or more commits is/are missing a Signed-off-by. Add it with 'git commit -s'."
-  return 1
+  test "$(git log --oneline --no-merges origin/"${ghprbTargetBranch}"..HEAD | wc -l)" -ne "$(git log --no-merges origin/"${ghprbTargetBranch}"..HEAD | grep -c Signed-off-by)" && echo "One or more commits is/are missing a Signed-off-by. Add it with 'git commit -s'." && return 1
```

Comment:
Can you explicit where ghprbTargetBranch is coming from ?

Reply:
Done.